policyengine_us/parameters/gov/states/id/tax/income/deductions/deductions.yaml

```
@@ -0,0 +1,9 @@
+description: Idaho accounts for these deductions of the agjusted gross income.
```

Though is this true? Hard to tell from this reference. Is it the list of itemized deductions? Does Idaho describe subtractions from AGI as "deductions"? (Many states call them subtractions, where deductions factor into taxable income instead). Does Idaho have a standard deduction?
```suggestion
description: Idaho deducts these items when calculating state adjusted gross income.
```